src/components/utilities/languages/English.json

```diff
@@ -70,6 +71,10 @@
   "CALCULATE CARBON FOOTPRINT": "CALCULATE CARBON FOOTPRINT",
 
   "METRIC TONS": "METRIC TONS",
+  "tons of CO2": "tons of CO2",
+  "Emissions Statement": "Your flight's CO2 emission will be neutralized in",
```

Comment:
do string to direct string so "Your flight's CO2 emission will be neutralized in": "Your flight's CO2 emission will be neutralized in"